sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/test/java/com/azure/data/schemaregistry/apacheavro/TestUtil.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.schemaregistry.apacheavro;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.test.InterceptorManager;
+import com.azure.core.test.utils.MockTokenCredential;
+import com.azure.identity.AzurePowerShellCredentialBuilder;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+/**
+ * Utility class for tests.
+ */
+public class TestUtil {
+
+    /**
+     * Gets a token credential for use in tests.
+     * @param interceptorManager the interceptor manager
+     * @return the TokenCredential
+     */
+    public static TokenCredential getTestTokenCredential(InterceptorManager interceptorManager) {
+        if (interceptorManager.isLiveMode()) {
+            return new AzurePowerShellCredentialBuilder().build();
+        } else if (interceptorManager.isRecordMode()) {
+            return new DefaultAzureCredentialBuilder().build();
+        } else {
+            return getPlaybackTokenCredential();
```

Comment:
nit: this could just be inlined

```suggestion
            return new MockTokenCredential();
```